### ISSUE TITLE
Make JSON output available via the Python API

### ISSUE THREAD

**Issue by nabla-c0d3** [nabla-c0d3/sslyze#439]
```
        for scan_command in ScanCommandsRepository.get_all_scan_commands():
            ScanCommandsRepository.get_implementation_cls(
                scan_command
            ).cli_connector_cls.register_json_serializer_functions()

[...]

json.dumps(asdict(scan_result), cls=_CustomJsonEncoder) 
```

**Comment by nabla-c0d3** [comment]
Released with v3.0.6.

**Comment by nabla-c0d3** [comment]
Documentation at https://nabla-c0d3.github.io/sslyze/documentation/running-scan-commands.html#exporting-to-json
